generated/nidaqmx/nidaqmx_library.h

```diff
@@ -17,7 +17,7 @@ namespace nidaqmx_grpc {
 class NiDAQmxLibrary : public nidaqmx_grpc::NiDAQmxLibraryInterface {
  public:
   NiDAQmxLibrary();
-  NiDAQmxLibrary::NiDAQmxLibrary(std::shared_ptr<nidevice_grpc::SharedLibraryInterface>);
+  NiDAQmxLibrary(std::shared_ptr<nidevice_grpc::SharedLibraryInterface>);
```

Comment:
Two minor nitpicks I didn't notice before:
- The parameter is unnamed
- This single-parameter constructor is missing `explicit`, so it enables implicit conversion from `shared_ptr<SharedLibraryInterface>` to `NiDAQmxLibrary&`, etc.

Reply:
>The parameter is unnamed

Is it not common to just specify parameter types in the function declaration?
I can add the name.

>This single-parameter constructor is missing explicit, so it enables implicit conversion from shared_ptr<SharedLibraryInterface> to NiDAQmxLibrary&, etc.

Nice catch.

Reply:
> Is it not common to just specify parameter types in the function declaration?

I usually see unnamed parameters used to work around unused variable warnings.